init.lua

```diff
@@ -38,7 +41,12 @@ beerchat = {
 
 }
 
+if nil == beerchat.main_channel_name or "" == beerchat.main_channel_name then
+	beerchat.main_channel_name = "main"
+end
```

Comment:
server would not crash with empty string but it should not be permitted. This way server admin will notice faster that there is mistake in minetest.conf

Reply:
there's probably some specific reason why empty string should not be permitted? Can it work with empty channel name?

I could see myself using empty string as main channel name if it is possible...

Reply:
theoretically it could work in this case.
t = {}; t[''] = 'bar'; print(t['']) --> bar
but you couldn't use dot syntax like this
t = {}; t['foo'] = 'bar'; print(t.foo) --> bar

Reply:
there are other numerous places that would need fixing as when command is given regex would be more complicated to detect switching to channel with zero-length string: '#main' works but what is '#'
going to give?

Reply:
would have to be '#:' as is supposed to work with channel names that have spaces in them:
'#my long channel name:'
Have never tried if that actually works and it must be hard to explain to users.

Reply:
Seems that channel names in chat are show as |#name| so empty would be |#| and I could easily see that as "main" chat channel name. So that's main thing it would give.
However there might be problems with channel switching if plain # wont work for switching to empty channel name.

Btw, I can create channel with spaces in name. But I cannot join that channel with normal `#channel name` command. Instead I had to join with `/force2channel My channel name with spaces, SX` :grinning: 

Did not yet find any way to create channel with empty name.

Reply:
no, have not tried names with spaces.
As much as I like the idea of empty channel name, I don't think it's good UX for less than geeky users.

Displaying nothing for main channel is easily implemented, but again much explaining needed to the kids. They have a hard time grasping the concept as it is.

Reply:
Let's do this in separate step as it's not related to what this request is adding ;D